app/views/benefits/CarFuelBenefitsAmountView.scala.html

```diff
@@ -0,0 +1,72 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.headerFooterTemplate.Layout
+@import views.html.templates.helpers.InputText
+@import views.html.templates.helpers.Heading
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.ErrorSummary
+@import utils.ViewUtils._
+
+@this(layout: Layout, formWithCsrf: FormWithCSRF, inputText: InputText, heading: Heading, button: Button, errorSummary: ErrorSummary)
+
+@(taxYear: Int, form: Form[BigDecimal], empId: String, previousAmount: Option[BigDecimal])(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+
+@titleText = @{
+        messages(s"benefits.carFuelAmount.title.${if(user.isAgent) "agent" else "individual"}")
+}
+
+@headingForUse = @{
+        messages(s"benefits.carFuelAmount.title.${if(user.isAgent) "agent" else "individual"}")
+}
+
+@paragraphText = @{
+        messages(s"benefits.carFuelAmount.paragraph.${if (user.isAgent) "agent" else "individual"}")
+}
+
+@optionalText = @{
+    if(previousAmount != None){messages("common.ifItWasNot", bigDecimalCurrency(previousAmount.get.toString))}
+}
+
+@contentHtml = {
+    @if(previousAmount != None){<p class="govuk-body">@optionalText</p>}
+    <p class="govuk-body">@paragraphText</p>
+}
+
+@layout(pageTitle = titleText, taxYear = Some(taxYear), hasErrors = form.hasErrors) {
+
+@errorSummary(form.errors)
```

Comment:
Hmm thats interesting, i've just done the same check on the interest/dividends/gift aid pages and they all seem to do the same as the above. Im wondering if we need a team char/ticket to check this, as i'm guessing it will be something to do with our generic input field helper